### ISSUE TITLE
Remote Input does not work on ChromeOS.

### ISSUE THREAD

**Issue by toaster42** [bk138/droidVNC-NG#211]
If you'd like to put out an incentive for fixing this bug, you can do so at https://issuehunt.io/r/bk138/droidVNC-NG

**Describe the bug**
When running on ChromeOS, and all permissions and the accessibility tool enabled, remote input fails to  elicit a response. 

**To Reproduce**
1. On a CrOS device in developer mode, install droidVNC-NG.
2. Run the server with all permissions and the accessibility tool enabled.
3. Connect to the server from another device and attempt mouse, keyboard, or touch input. 

**Expected Behavior**
If it is technically feasible, it would be lovely for the VNC server to allow for full remote desktop behavior (ie. mouse and keyboard work remotely)
 
**Logs/Backtraces**
N/A

**Your environment (please complete the following information):**
 - droidVNC-NG version: 2.4.0
 - Android version: Android 11 (R125-15853.53.0 release-keys) on CrOS (125.0.6422.140 (Official Build) (64-bit))
 - Client-side OS and version: Android 14
 - VNC client and version: MultiVNC

**Additional context**
I realize that this use case is likely outside the imagined scope for this app.  That said, there is a total lack of decent remote desktop solution for ChromeOS, so if this could be extended to that market, it would be greatly appreciated by a fair number of people.

**Comment by bk138** [comment]
Thanks for reporting! Few questions:

- are you taken to the accessibility settings screen to enable droidVNC-NG there?
- do hotkeys like Home etc show any response?

**Comment by glaucors** [comment]
Hello, I have the same problem, I believe it is not accepting gestures.
Equipment: TV BOX Aquário STV-3000 Plus
Android TV OS Version 11
DroidVNC-NG 2.5.0

> Are you taken to the accessibility settings screen to enable droidVNC-NG there?
> *       No, you need to find it manually

>Ao hotkeys like Home etc show any response?
>*       Yes, the home hotkeys.



![AndroidBOX - Info](https://github.com/user-attachments/assets/eff87167-1251-482e-b47f-e638c6991d75)
![DoidVNC-NG](https://github.com/user-attachments/assets/62999d4e-2176-4343-8f0a-85eef6f43c89)
![DoidVNC-NG-accessibility](https://github.com/user-attachments/assets/810cde30-c66f-4089-9199-8800ae6b72f8)

**Comment by virtimus** [comment]
As additional effect  - in ChromeOS (connecting from vnc client) I can control "Play Store" window but not main desktop nor ie Chrome Window... probably those apps not suporting Android accessibility api ...
